### PR TITLE
Auto scaler profile fixes

### PR DESCRIPTION
**Pull request by oleksiimorozenko**
This PR fixes the mismatch of the types of the dynamic `auto_scaler_profile` object

### CODE REVIEW

**Comment by oleksiimorozenko** [comment]
Oh sorry, it was accidentally

**Comment by oleksiimorozenko** [comment]
I suppose it would be better to merge after merging [the previous PR](https://github.com/claranet/terraform-azurerm-aks/pull/9)

File: variables.tf [hunk 1]
```diff
@@ -196,26 +195,26 @@ variable "aci_subnet_id" {
...
+    scale_down_delay_after_delete    = optional(string)
+    scale_down_delay_after_failure   = optional(string)
+    scan_interval                    = optional(string)
+    scale_down_unneeded              = optional(string)
+    scale_down_unready               = optional(string)
+    scale_down_utilization_threshold = optional(number)
+    empty_bulk_delete_max            = optional(number)
+    skip_nodes_with_local_storage    = optional(bool)
+    skip_nodes_with_system_pods      = optional(bool)
+  })
```
Comment (jmapro) [inline variables.tf:217]: Why removing all default values ? I think it's important to provide some default values applicable to most of workloads

**Review by jmapro — CHANGES_REQUESTED** [review]

**Review by jmapro — APPROVED** [review]
LGTM

**Pull request merged**